src/PDFTemplates/ExecutiveReport.tsx

```diff
@@ -0,0 +1,615 @@
+            y: oversizedCount,
+            percentage: oversizedPercentage
+        },
+        {
+            x: 'Idling',
+            y: idlingCount,
+            percentage: idlingPercentage
+        },
+        {
+            x: 'Waiting for data',
+            y: waitingForDataCount,
+            percentage: waitingForDataPercentage
+        }
+    ];
+
+    const performanceData = [
+        {
+            x: 'Disk IO',
+            y: ioCount
+        },
+        {
+            x: 'RAM',
+            y: memoryCount
+        },
+        {
+            x: 'CPU',
+            y: cpuCount
+        }
+    ];
+
+    const instanceTableDetails = [
+        {
+            id: 'current_instance_types',
+            heading: 'Most used current instance types',
+            description: 'We have identified these instance types based on the data from the fresh systems.',
+            data: currentData
+        },
+        {
+            id: 'suggested_instance_types',
+            heading: 'Most suggested instance types',
+            description: 'We are suggesting these instance types based on the data from the fresh systems.',
+            data: suggestedData
+        },
+        {
+            id: 'historical_instance_types',
+            heading: 'Most suggested instance types (45 days)',
+            description: 'In the last 45 days we suggested you these instances # of times.',
+            staleDescription: 'Some of the instances are now identified as stale (more than 7 days not reporting data).',
+            data: historicalData
+        }
+    ];
+
+    const blueColorScale = getThemeColors(ChartThemeColor.blue).pie?.colorScale;
+    const breakdownColors = getThemeColors(ChartThemeColor.multiOrdered).pie?.colorScale;
+    return (
+        <Fragment>
+            <PageBreak>
+                <Title
+                    headingLevel="h1"
+                    className="pf-v5-u-danger-color-100 pf-v5-u-font-size-4xl pf-v5-u-mb-lg"
+                >
+                  Resource optimization service report
+                </Title>
+                <Stack>
+                    <StackItem>
+                        <TextContent>
+                            <Text className="pf-v5-u-font-size-xs">
+                              This executive summary highlights the performance for your
+                              registered systems included in the resource optimization
+                              service.
+                            </Text>
+                        </TextContent>
+                    </StackItem>
+                    <StackItem>
+                        <Title
+                            headingLevel="h5"
+                            size="md"
+                            style={{
+                                marginTop: '6px',
+                                color: 'var(--pf-global--danger-color--100)'
+                            }}
+                        >
+                          Registered systems
```

Comment:
@Hyperkid123 based on the Pdf which you shared on slack thread there is minor fixing needed in the title (it is needed in red color) and can you please reduce spacing in the `System performance issues` table. Sharing screenshot for your reference (left one is from production envrionment last week and right one is what you shared)
![Uploading image.png…]()